packaging/templates/mainnet-v1/sentry/sentry/bor/pbss_config.toml

```diff
@@ -1,15 +1,16 @@
-chain = "mumbai"
+chain = "mainnet"
 # identity = "node_name"
 # verbosity = 3
 # vmdebug = false
 datadir = "/var/lib/bor/data"
 # ancient = ""
-# "db.engine" = "leveldb"
+db.engine = "pebble"
+state.scheme = "path"
```

Comment:
Both the flags should be wrapped in quotes as mentioned above.

Reply:
Updated